aplib.net-demo/Assets/Scripts/Entities/Weapons/MeleeWeapon.cs

```diff
@@ -33,7 +33,7 @@ public class MeleeWeapon : Weapon
     /// Ensure the height is at least twice the radius, because the height of the hitzone (capsule) must at least be the diameter of the spheres.
     /// (If the two spheres are at the same position, the capsule just becomes a sphere and height = 2 * radius)
     /// </summary>
-    private void Awake()
+    void Start()
```

Comment:
Why did you change this to void Start with no access modifier (probably accidental)?

Reply:
this used to override a start method, private is not necessary as the default is private

Reply:
The rest of the code (even your own) adds private keywords to `Awake `and `Start `though, so I would still add it for consistency.